src/azure-cli/azure/cli/command_modules/appconfig/commands.py

```diff
@@ -50,6 +51,11 @@ def load_command_table(self, _):
         client_factory=cf_replicas
     )
 
+    configstore_snapshot_util = CliCommandType(
+        operations_tmpl='azure.cli.command_modules.appconfig.snapshot#{}',
+        table_transformer=configstore_snapshot_output_format
+    )
```

Comment:
How come this one doesn't have client factory like all the others?

Reply:
The client factories create a Python SDK client for the given command. Since there is no SDK client implementation for snapshots yet, the client is implemented in `_snapshot_custom_client.py` so this is directly initialized where it is used. This will be updated to include the corresponding client factory when available.